File: app/src/main/java/com/vladan/color_interview/model/ApiResponsePerson.java
package com.vladan.color_interview.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by vladan on 8/28/2020
 */
public class ApiResponsePerson {
    @SerializedName("status")
    public String status;

    @SerializedName("data")
    public Person data;

}

File: app/src/main/java/com/vladan/color_interview/model/ApiResponsePerson.kt
package com.vladan.color_interview.model

import com.google.gson.annotations.SerializedName

/**
 * Created by vladan on 8/28/2020
 */
class ApiResponsePerson {
    @SerializedName("status")
    var status: String? = null

    @JvmField
    @SerializedName("data")
    var data: Person? = null
}